docker/opbeans/node/entrypoint.sh

```diff
@@ -1,11 +1,14 @@
 #!/usr/bin/env sh
 set -ex
 if [ -f /local-install/package.json ]; then
-    echo "Installing from local folder"
-    # copy to folder inside container to ensure were not poluting the local folder
-    cp -r /local-install ~
-    cd ~/local-install && npm install .
-    cd -
+    echo "Installing elastic-apm-node from local folder (--opbeans-node-agent-local-repo)"
+    # Copy to a folder inside container to ensure we're not polluting the
+    # local folder. Skip possibly huge dirs to speed this up.
+    apk add rsync
```

Comment:
Is installing rsync appropriate here?  Would doing so in the Dockerfile be more appropriate?

Reply:
it is better to add packages on the Dockerfile